_includes/page-information.liquid

```diff
@@ -2,7 +2,7 @@
 
   {% include metadata.liquid %}
 
-  {% if page.toc == true and site.toc == true %}
+  {% if page.toc != false and site.toc == true or page.toc == true %}
```

Comment:
I'd like to add some parenthesis to make the order of evaluation of the "and" and the "or" obvious.

Is it (page.toc != false and site.toc == true) OR (page.toc == true) ?
Is it (page.toc != false) AND (site.toc == true or page.toc == true) ?

Reply:
As this is essentially a partial revert, I suggest merge as-is, and address the parens separately.

https://github.com/publiccodenet/jekyll-theme/pull/45/files#diff-3eb3545daec35524cc5339a5fd78e5eb40cd85cbfa0ba5d734e4f47833174c31L5-R5

Reply:
In this case, I think the two are the same. However, I agree that it should be written a bit more clearly.

Evaluation:

1. If page doesn't want a table of contents, stop here
2. If site wants a table of contents, insert it
3. If page wants a table of contents, insert it

Is this something to add to the issue and reopen it?